website/static/js/contribAdder.js

```diff
@@ -12,6 +12,8 @@ var oop = require('js/oop');
 var $osf = require('js/osfHelpers');
 var osfLanguage = require('js/osfLanguage');
 var Paginator = require('js/paginator');
+var NodeSelectTreebeard = require('js/nodeSelectTreebeard');
+var m = require('mithril');
 
 var NODE_OFFSET = 25;
```

Comment:
This NODE_OFFSET is used for indent original component tree. Since you are using the new one, this is no longer necessary.